### ISSUE TITLE
Incorrect signature of pushState and replaceState methods in lib.d.ts

### ISSUE THREAD

**Issue by griii2** [Microsoft/TypeScript#10915]
**TypeScript Version:**  1.8.10 

**Code**

```ts

    window.history.pushState(statedata);
    window.history.replaceState(statedata);

```

**Expected behavior:**

> Uncaught TypeError: Failed to execute 'pushState' on 'History': 2 arguments required, but only 1 present.
> Uncaught TypeError: Failed to execute 'replaceState' on 'History': 2 arguments required, but only 1 present.

According to https://html.spec.whatwg.org/multipage/browsers.html only the last parameter is optional and Chrome throws an error at runtime.

**Actual behavior:** 
No error, there are two optional parameters in lib.d.ts:

```ts
interface History {
    pushState(statedata: any, title?: string, url?: string): void;
    replaceState(statedata: any, title?: string, url?: string): void;
}
```

**Comment by mhegazy** [comment]
PRs welcomed. You can find more information about contributing lib.d.ts fixes at https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes.

**Comment by YuichiNukiyama** [comment]
This issue already has been fixed. see [Fix the History api #142](https://github.com/Microsoft/TSJS-lib-generator/pull/142).
